custom/ewsghana/reports/specific_reports/stock_status_report.py

```diff
@@ -297,11 +285,23 @@ def charts(self):
 class StockoutTable(EWSData):
 
     slug = 'stockouts_product_table'
-    title = 'Stockouts'
     show_chart = False
     show_table = True
 
     @property
+    def title(self):
+        if not self.location:
+            return ""
+
+        location_type = self.location.location_type.name.lower()
+        if location_type == 'country':
+            return "Stockouts - CMS, RMS, and Teaching Hospitals"
+        elif location_type == 'region':
+            return "Stockouts - RMS and Teaching Hospitals"
+        elif location_type == 'district':
+            return "Stockouts"
```

Comment:
I think this suffers from the same problem as other reports (if the `location_type` is not expected it will just return `None`. Is that intentional? It seems like this could just be an `else`